service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerTest.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(RespondentController.class)
+@OverrideAutoConfiguration(enabled = true)
+class RespondentControllerTest extends AbstractControllerTest {
+
+    RespondentControllerTest() {
+        super("enter-respondents");
+    }
+
+    private static final String ERROR_MESSAGE = "Date of birth cannot be in the future";
+
+    @Test
+    void aboutToStartShouldPrepopulateRespondent() {
+        CaseDetails caseDetails = CaseDetails.builder()
+            .data(Map.of("data", "some data"))
+            .build();
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = postAboutToStartEvent(caseDetails);
+
+        assertThat(callbackResponse.getData()).containsKey("respondents1");
+    }
+
+    @Test
+    void shouldReturnDateOfBirthErrorsForRespondentWhenFutureDateOfBirth() {
+        CallbackRequest request = CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .id(12345L)
+                .data(ImmutableMap.of(
+                    "respondents1", ImmutableList.of(
+                        ImmutableMap.of(
```

Comment:
As discussed the all Callbackrequest can be omitted now (see other tests!) that's at least some simplification.